.gitignore

```diff
@@ -284,3 +284,5 @@ dmypy.json
 # End of https://www.gitignore.io/api/emacs,python,intellij+all
 glove_embeddings
 /output/
+/tests/resources/test_adjectives.prepro
```

Comment:
will we need these files for tests to run in CI? I thought so?

Reply:
these are created during the test runs and then deleted, so I might as well take them out of the gitignore not to confuse anyone